specification/kubernetesconfiguration/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,13 @@
+## TypeScript
```

Comment:
I think what bicep team wants is the readme.md file only, and those readme.language.md files should be removed once you have already splited into different independent sub services.